src/data/courseDependenciesData.js

```diff
@@ -959,61 +959,34 @@ const courseDependencyData = (courseSlug) =>
             slug: 'the-beginner-s-guide-to-react',
           },
         ],
-        courseProject: {
-          label: 'Portfolio Project',
-          url:
-            '/projects/create-an-ecommerce-store-with-next-js-and-stripe-checkout',
-          text: `After this course, you'll be ready to start building your own e-commerce store. Need an idea? Start with this Project!
-      `,
-        },
         pairWithResources: [
+          {
```

Comment:
Scope the  `pairWithResources` to content that was specifically designed for this course.